packages/frontend-shared/src/gql-components/ChooseExternalEditorModal.cy.tsx

```diff
@@ -13,14 +13,20 @@ describe('ChooseExternalEditor', { viewportHeight: 400, viewportWidth: 300 }, ()
         ]
       },
       render: (gql) => {
-        return <div class="p-16px"><ChooseExternalEditor gql={gql} /></div>
+        return <div class="p-16px"><ChooseExternalEditorModal gql={gql} open /></div>
       },
     })
 
+    cy.contains(defaultMessages.globalPage.externalEditorPreferences)
+    cy.contains(defaultMessages.globalPage.externalEditorPreferencesDescription1)
+    cy.contains('You can change this later in the Settings page.')
```

Comment:
More of a question for me than for you -- when coming across hardcoded strings that are not in `defaultMessages`, is the expectation to leave as is or to update `defaultMessages`?

Reply:
Ah I scrolled on further down and I see 
```
    "externalEditorPreferencesDescription2": "You can change this later in the {0} page.",
```
Is it not usable in the test b/c of the `{0}` variable?

Reply:
I think I may need to update it, found another test where we are using something like `defaultMessages.globalPage.externalEditorPreferencesDescription1.replace('{0}', 'Settings')` and it should work! good catch!